apps/codeforafrica/src/pages/api/admin/config.js

```diff
@@ -281,5 +342,84 @@ module.exports = {
         },
       ],
     },
+    {
+      name: "team",
+      label: "Data | Team",
+      label_singular: "Team Member",
+      folder: "content/team",
+      create: true,
+      identifier_field: "name",
+      fields: [
+        {
+          label: "Id",
+          name: "id",
+          widget: "uuid",
+        },
+        {
+          label: "Name",
+          name: "name",
+          widget: "text",
+        },
+        {
+          label: "Title",
+          name: "title",
+          widget: "markdown",
+        },
+        {
+          label: "Description",
+          name: "description",
+          widget: "markdown",
+        },
+        {
+          label: "Image",
+          name: "image",
+          widget: "image",
+        },
+        {
+          label: "Projects",
+          name: "projects",
+          widget: "relation",
+          label_singular: "Project",
```

Comment:
For obvious cases, NetlifyCMS handles label singular well enough.

Reply:
In this case since the `Label` was updated to `Data | Projects` it isn't able to translate it to just `Project`